dr/firmware/ucf/vsim.tcl

```diff
@@ -1,3 +1,8 @@
 set_property SOURCE_SET sources_1 [get_filesets sim_1]
 add_files -fileset sim_1 -norecurse ../emData/in.txt
 add_files -fileset sim_1 -norecurse ../emData/out.txt
+add_files -fileset constrs_1 -norecurse ../../src/l1tk-for-emp/dr/firmware/ucf/constraints.xdc
```

Comment:
please change to global constraint file moved to top

Reply:
Done :-)